alembic/env.py

```diff
@@ -95,6 +130,23 @@ def run_migrations_offline() -> None:
 
     with context.begin_transaction():
         context.run_migrations()
+        with context.begin_transaction():
+            print("-- Dropping views")
+            print(f"DROP VIEW IF EXISTS {schema_name}.visit1;")
+            print(f"DROP VIEW IF EXISTS {schema_name}.ccdvisit1;")
+
+            context.run_migrations()
+
+            print(f"""
+                CREATE VIEW {schema_name}.ccdvisit1
+                    AS SELECT * FROM {schema_name}.ccdexposure;
+                ALTER TABLE {schema_name}.ccdvisit1
+                    RENAME COLUMN ccdexposure_id TO ccdvisit_id;
+                CREATE VIEW {schema_name}.visit1
+                    AS SELECT * FROM {schema_name}.exposure;
+                ALTER TABLE cdb_{schema_name}.visit1
+                    RENAME COLUMN exposure_id TO visit_id;
+                """)
```

Comment:
I'm a little confused: since this is outside `run_migrations()`, it runs on both upgrade and downgrade?